svgnative/cmake/FindSkia.cmake

```diff
@@ -0,0 +1,88 @@
+# - Try to find the Skia library to be linked with,
+# and prepare a few variables for linker flags.
+#
+# ----------------------------------------------------------------------------
+#
+# Copyright (C) 2020 suzuki toshiya <mpsuzuki@hiroshima-u.ac.jp>
+#
+# Redistribution and use in source and binary forms, with or without
```

Comment:
Do you mind putting this under Apache License?

Reply:
OK!